pkg/core/fixtures/unsafe/main.go

```diff
@@ -0,0 +1,21 @@
+package plugin
```

Comment:
And here the package should be `unsafe`, no?

Reply:
It's not mandatory, but plugin often use a "plugin" package ( like the main package )